contracts/RelayHub.sol

```diff
@@ -94,7 +95,7 @@ contract RelayHub is IRelayHub {
         uint256 amount = msg.value;
         require(amount <= maximumRecipientDeposit, "deposit too big");
 
-        balances[target] = SafeMath.add(balances[target], amount);
+        balances[target] = balances[target].add(amount);
```

Comment:
not really needed just after a check that it is actual below maxDeposit...

Reply:
It is not about being needed